### ISSUE TITLE
we have a regression that riot/web now queries AS room directories as well as the main HS one, completely changing our carefully curated roomdir for matrix.org

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#9285]
unless someone changed the config.json to do this, but I do not think it is desirable behaviour. We should only be showing the matrix.org rooms by default, not matrix.org+freenode+snoonet+everything else in the directory

**Comment by jryans** [comment]
Hmm, so the default is meant to be the option that says "[m] Matrix" underneath the default HS, so that only native Matrix rooms are shown?

![image](https://user-images.githubusercontent.com/279572/55022634-e38f2580-4ff2-11e9-8e09-d8782c426ee2.png)

(Currently the default option is "matrix.org", which includes all networks on the HS.)

**Comment by lampholder** [comment]
I can confirm the behaviour used to be to show the default server's `[m] Matrix` subsection.

There's so much wrong with the room directory UX, but I'd suggest just putting this back how it was for now.

**Comment by lampholder** [comment]
fire label applied because this is affecting users' first experience of the app

**Comment by ara4n** [comment]
this isn't fixed; by defaulting the directory to show all the random bridged rooms too, the native matrix ones get drowned out. the intention was always to just show the native matrix ones by default to avoid weird bridged rooms coming top of the list.

**Comment by turt2live** [comment]
@ara4n the fix hasn't landed on /app yet, it's stuck on /develop

**Comment by turt2live** [comment]
I think I found the one spot it doesn't work in, and it's a feature as far as I can tell. Been around since The Great Relayering at a minimum, and working as intended. I have opened a PR to not include all networks though as that seems more desirable. 

Only appears to happen when clicking the server name. Note that changing this breaks the hierarchy of the dropdown, although that hierarchy is confusing: https://github.com/vector-im/riot-web/issues/6260
